spec/components/header_component_spec.rb

```diff
@@ -8,6 +8,8 @@
   it 'renders the header' do
     render_inline(component)
 
-    expect(page).to have_css('header .masthead .h1', text: 'Argo')
```

Comment:
Oddly, I'm finding these kind of expects failing more and more without doing a `within` for some reason.

Reply:
I don't think `within` works as expected in component specs. I suspect there is some other problem.